treeherder/webapp/api/bugzilla.py

```diff
@@ -51,8 +52,20 @@ def create_bug(self, request):
             'description': description,
             'comment_tags': "treeherder",
         }
-        if len(params.get('groups')) > 0:
-            data['groups'] = params.get('groups')
+        if params.get("is_security_issue"):
+            security_group_list = list(
+                BugzillaSecurityGroup.objects.filter(product=data.get("product")).values_list(
+                    "security_group", flat=True
```

Comment:
I'm curious about this approach. Is there a reason why we couldn't fetch the security info in the UI as needed, do filtering there and pass the security product group as a param to the API? Why would we need to run a crob job to periodically store the data?

Reply:
It's a big flow and slow to fetch, so I took this approach. Filing a bug from Treeherder has a synchronous workflow: user submits from UI to server, server submits to Bugzilla, Bugzilla sends bug number to server, server sends bug number to client-side UI and sets the bug as classified.